reflex-dom-core/src/Reflex/Dom/Attributes/Class.hs

```diff
@@ -0,0 +1,108 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | Functions for adjusting parts of the class attribute.
+module Reflex.Dom.Attributes.Class
+  (
+  -- * Declaratively specifying classes (for static or dynamic attributes)
+    declareClass
```

Comment:
Using `declareClass`, etc., means that for someone who wants to do something basic, they now need to have memorized which attributes ought to be done with `declare` and which ought to be done with strings.  It's not clear to me how they would discover this, either.